networks/twatter.py

```diff
@@ -76,61 +76,59 @@ class Twitter(Network):
 
     @taskify
     async def check_tweets(self):
-        """ Parse our stupid homebrew conf file """
 
+        info('checking tweets')
         if not self._twitter_conf:
             warning('_twitter_conf not (yet?) loaded!')
-
             return
 
-        for serv in self._twitter_conf.keys():
-            debug(serv)
+        info('parsing twitter_conf')
+        tooters = self._twitter_conf
 
-            for chann in self._twitter_conf[serv].keys():
-                debug(chann)
+        for tooter in tooters:
```

Comment:
one of the key edge cases to think about is retweeting the same account in multiple channels. 

as it stands, it will run `get_user_timeline` for the same user, for each channel. :(